oap-mail/src/main/java/oap/mail/MailQueue.java

```diff
@@ -15,7 +17,9 @@
     private final Path location;
 
     public MailQueue( Path location ) {
-        this.location = location.resolve( "mail.gz" );
+        this.location = Optional.ofNullable( location )
+            .orElse( Paths.get( "/tmp/mail.gz" ) )
```

Comment:
the idea here is when location is not supplied - email is not persistent. It's useful for tests, or rare cases when email queue should not survive reboots